src/app/components/AdvocatesTable.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import { Advocate } from "@/app/types";
+
+type AdvocatesTableProps = {
+  advocates: Advocate[];
+};
+
+const AdvocatesTable: React.FC<AdvocatesTableProps> = ({ advocates }) => (
+  <div className="overflow-x-auto">
+    <table className="min-w-full bg-white shadow-md rounded-lg overflow-hidden">
+      <thead className="bg-blue-600 text-white">
+        <tr>
+          <th className="py-3 px-4 text-left font-semibold">First Name</th>
+          <th className="py-3 px-4 text-left font-semibold">Last Name</th>
+          <th className="py-3 px-4 text-left font-semibold">City</th>
+          <th className="py-3 px-4 text-left font-semibold">Degree</th>
+          <th className="py-3 px-4 text-left font-semibold">Specialties</th>
+          <th className="py-3 px-4 text-left font-semibold">Experience</th>
+          <th className="py-3 px-4 text-left font-semibold">Phone Number</th>
+        </tr>
+      </thead>
+      <tbody>
+        {advocates.map((advocate, idx) => (
+          <tr
+            key={advocate.id ?? idx}
```

Comment:
React requires each element in a list to have a unique `key` prop when rendering multiple items dynamically.

Adding this key attribute gets ride of the console.error:

![Screenshot 2025-04-29 at 6 23 49 PM](https://github.com/user-attachments/assets/3a484d9c-3a75-42c9-8769-e7bcf78eff0f)